nearest.go

```diff
@@ -0,0 +1,52 @@
+package osrm
+
+import geo "github.com/paulmach/go.geo"
+
+// NearestRequest represents a request to the nearest method
+type NearestRequest struct {
+	Profile  string
+	GeoPath  GeoPath
+	Bearings []Bearing
+
```

Comment:
Extra minor: could you please remove extra space here? Just to be a bit more consistent with other requests around

Reply:
done